packages/autorest.typescript/test/integration/generated/appconfiguration/package.json

```diff
@@ -18,17 +18,12 @@
   "types": "./types/appconfiguration.d.ts",
   "devDependencies": {
     "@microsoft/api-extractor": "^7.31.1",
-    "@rollup/plugin-commonjs": "^24.0.0",
-    "@rollup/plugin-json": "^6.0.0",
-    "@rollup/plugin-multi-entry": "^6.0.0",
-    "@rollup/plugin-node-resolve": "^13.1.3",
     "mkdirp": "^2.1.2",
-    "rollup": "^2.66.1",
-    "rollup-plugin-sourcemaps": "^0.6.3",
     "typescript": "~5.0.0",
     "uglify-js": "^3.4.9",
     "rimraf": "^5.0.0",
-    "dotenv": "^16.0.0"
+    "dotenv": "^16.0.0",
+    "@azure/dev-tool": "^1.0.0"
```

Comment:
Could you help understand why this change happens in `not azureSDKForJs` env?

https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/test/commands/test-swagger-gen.ts#L35-L45